pkg/maps/lbmap/ipv4.go

```diff
@@ -222,6 +223,8 @@ func (s *Service4Value) GetCount() int        { return int(s.Count) }
 func (s *Service4Value) SetRevNat(id int)     { s.RevNat = uint16(id) }
 func (s *Service4Value) GetRevNat() int       { return int(s.RevNat) }
 func (s *Service4Value) RevNatKey() RevNatKey { return &RevNat4Key{s.RevNat} }
+func (s *Service4Value) SetFlags(flags uint8) { s.Flags = flags }
+func (s *Service4Value) GetFlags() uint8      { return s.Flags }
```

Comment:
Could this return a `type ServiceFlags uint8` which implements `Stringer` to get human-readable flags?